graphql-cli/.gitignore

```diff
@@ -0,0 +1,38 @@
+# Compiled class file
+*.class
+
+# Log file
+*.log
+
+# BlueJ files
+*.ctxt
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
+
+# Package Files #
+*.war
+*.nar
+*.ear
+*.zip
+*.tar.gz
+*.rar
+
+# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+hs_err_pid*
+
+#Ballerina configuartion file
+ballerina.conf
+Config.toml
+
+#json files created inside resources folder
+**/resources/*.json
+
+# IDEA file
+.idea
+
+# Ignore Gradle project-specific cache directory
+.gradle
+
+# Ignore Gradle build output directory
+build
```

Comment:
We already have a `gitignore` file in root level.
Shall we utilise that one file instead of having multiple `gitignore` files in different places?

Reply:
This is resolved now & changes are amended. Thank you.